portfolio/src/main/webapp/script.js

```diff
@@ -81,8 +81,29 @@ function hasWon(){
     }
 }
 
-async function getHello() {
+async function getComments() {
   const response = await fetch('/data');
-  const hello = await response.text();
-  document.getElementById('hello-container').innerText = hello;
-}
\ No newline at end of file
+  const comments = await response.text();
+  document.getElementById('comments-container').innerText = comments;
+}
+
+/**
+ * Fetches the current state of the game and builds the UI.
+ */
+async function getCommentSection() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+
+  // Build the list of history entries.
```

Comment:
nit: these lines 96-99 should be indented another 2 spaces, so that it's clear all of this code runs after the `fetch('/data')`

Reply:
fixed